nvi-evaluator/src/main/java/no/sikt/nva/nvi/evaluator/calculator/CandidateCalculator.java

```diff
@@ -139,24 +126,14 @@ private Map<URI, List<URI>> getVerifiedCreatorsWithNviInstitutions(JsonNode body
 
     private List<URI> getTopLevelNviInstitutions(JsonNode creator) {
         return streamNode(creator.at(JSON_PTR_AFFILIATIONS))
-                   .map(CandidateCalculator::extractId)
-                   .map(this::fetchOrganization)
+                   .map(JsonUtils::extractId)
+                   .map(organizationRetriever::fetchOrganization)
```

Comment:
It might be better to stream the TLIs to filter distinct before processing to remove the potential that the same institution is extracted twice